### PR TITLE
enh(config): add a field to set database root user at installation 

### PR DESCRIPTION
**Pull request by cgagnaire**
<h2> Description </h2>

Add a field to set database root user at installation if the root user is not actually root.

Closes: #6051 

<h2> Type of change </h2>

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [x] 19.10.x

<h2> How this pull request can be tested ? </h2>

Install a fresh 19.04.x instance.

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

<h5> Centreon team only </h5>

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests covers 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/install/steps/functions.php [hunk 1]
```diff
@@ -43,6 +43,10 @@ function getTemplate($dir)
  */
 function myConnect()
 {
+    $user = "root";
+    if (isset($_SESSION['root_user']) && $_SESSION['root_user']) {
```
Comment (sc979) [inline www/install/steps/functions.php]:
```suggestion
    if (!empty($_SESSION['root_user'])) {
```

File: www/install/steps/functions.php [hunk 2]
```diff
@@ -43,6 +43,10 @@ function getTemplate($dir)
...
+        $user = $_SESSION['root_user'];
+    }
     $pass = "";
     if (isset($_SESSION['root_password']) && $_SESSION['root_password']) {
```
Comment (sc979) [inline www/install/steps/functions.php]:
```suggestion
    if (!empty($_SESSION['root_password'])) {
```

**Comment by adr-mo** [comment]
I've force pushed the branch to rebase it on master for the **validation**

**Comment by ganoze** [comment]
Rebased branch on top of 19.10 (PR #7542, branch update-to-19.10).

**Pull request merged**